frontend/src/views/Game.tsx

```diff
@@ -34,10 +34,16 @@ import {
   connect, disconnect, routes, send, subscribe,
 } from '../api/Socket';
 import GameNotificationContainer from '../components/game/GameNotificationContainer';
+import { NoColorExternalLink } from '../components/core/Link';
 import Language from '../api/Language';
 
 const StyledMarkdownEditor = styled(MarkdownEditor)`
```

Comment:
So with reference to [this discussion](https://github.com/rocketden/main/pull/114#issuecomment-799796479), I saw that the game page has lines at those locations even when there isn't an overflow of users - is there a way to remove those lines if the overflow doesn't exist?

<img width="1050" alt="Shadow Remaining on Edges of Player List" src="https://user-images.githubusercontent.com/7987279/111438238-3de28e80-86c1-11eb-81f8-9c4f950e22d6.png">

Reply:
I played around with this a bit and it seems like it's a thin line that shows up every now and then on very specific browser widths. I couldn't come up with a perfect solution other than setting the background shadow position to like 0.1% and 99.9% so that the cover can completely hide it even with off by one browser pixels - not the best solution lol and it might not handle 100% of the unique browser widths but hopefully should capture the majority of them - lmk what you think

Reply:
Huh that's a really weird bug - yeah I'm ok with this solution, we can revisit this later if need be to come up with a more rigorous solution.